### PR TITLE
feat: About Page animation

### PR DESCRIPTION
**Pull request by arse0039**
Updated about page to have animation for when the avatar is clicked

### CODE REVIEW

File: frontend/src/components/AboutSection.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+interface AboutSectionProps {
+    name: string;
+    
+}
+
+const AboutSection: React.FC<AboutSectionProps> = ({name}) => {
+    const [visibleName, setVisibleName] = useState<string>('');
+
+    useEffect(() => {
+        setVisibleName('')
```
Comment (davidclaphan) [inline frontend/src/components/AboutSection.tsx:13]: this is awesome

File: frontend/src/components/TranslucentModalDiv.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React, {ReactNode} from 'react';
```
Comment (davidclaphan) [inline frontend/src/components/TranslucentModalDiv.tsx]: eslint gives me a warning for unused-vars

File: frontend/src/components/aboutModal.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```
Comment (davidclaphan) [inline frontend/src/components/aboutModal.tsx:1]: same linting error here

**Review by davidclaphan — COMMENTED** [review]
looking great

**Review by davidclaphan — DISMISSED** [review]

File: frontend/src/components/TranslucentModalDiv.tsx (hunk 2 above)
Comment (arse0039) [reply frontend/src/components/TranslucentModalDiv.tsx]: Good catch!

**Review by davidclaphan — APPROVED** [review]

**Pull request merged**